torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -142,6 +142,14 @@ RegisterOperators reg({
             return 0;
           };
         }),
+    Operator(
+        prim::NoneGenerator,
+        [](Node* node) {
+          return [](Stack& stack) {
+            stack.push_back(at::Tensor());
```

Comment:
Did you make sure the generated code will correctly pop this argument from the stack, even though it won't use it?

Reply:
I am pretty sure it does, because previously the schema had a tensor.